UploadServiceLogo/index.ts

```diff
@@ -27,6 +28,8 @@ const servicesContainer = database.container(SERVICE_COLLECTION_NAME);
 
 const serviceModel = new ServiceModel(servicesContainer);
 
+const blobService = createBlobService(config.StorageConnection);
```

Comment:
isnt this `AssetsStorageConnection`?

Reply:
Yes it is but it's not present in config module :) I'd rather add this variable.